impl/client/src/main/javascript/web/prompting/components/DojoDateTextBoxComponent.js

```diff
@@ -201,7 +201,7 @@ define(['cdf/components/BaseComponent', "dojo/date/locale", 'dijit/form/DateText
           var dojoFormat;
           var dateFormat = this._getDateFormat();
 
-          if(dateFormat.match(/(^|(?!y).)(y{1}(?!y))/)) {
+          if(dateFormat.match(/o/)) {
```

Comment:
@tmorgner https://api.jqueryui.com/datepicker/ 
I don't remember why, but jquery was the reason.